cmd/server.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"log"
+	"net"
+	"strconv"
+)
+
+const client_msg_buffer = 1024
+
+func main() {
+	var (
+		address = "localhost"
+		port    = "8080"
+	)
+
+	ln, err := net.Listen("tcp", address+":"+port)
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Println("Server ready for connections, address:", ln.Addr())
+	defer ln.Close()
```

Comment:
how about using channels for graceful shutdown?

Reply:
I think this is resolved now? (8a56c5e)